docs/tutorial/piezotracking.rst

```diff
@@ -0,0 +1,136 @@
+    baseline_2x.plot_residual(label="baseline f2x")
+    plt.legend(loc='lower right')
+
+.. image:: figures/piezotracking/baseline_residual.png
+
+The residuals should be distributed around zero and should ideally look like noise.
+
+Calculating the piezo distance and corrected force
+--------------------------------------------------
+
+When a tether is present, it exerts a force on the beads.
+This force results in the displacement of the beads from the trap centers.
+Assuming only small excursions from the trap center, we can assume this relation to be linear (proportional to the trap stiffness :math:`\kappa`).
+Therefore, we can compute the bead displacement :math:`\delta x` directly from the force signal.
+
+.. math::
+
+    \delta x_1 = \frac{F_{1x}}{\kappa_{1x}}
+
+This means that the surface-to-surface distance can be computed by correcting the calibrated distance that was based on the trap, with the force signals and their respective stiffnesses as follows.
+
+.. math::
+
+    d_{piezo} = d_{no\_tether}(x_{trap1}) - \frac{F_{1x}}{\kappa_{1x}} + \frac{F_{2x}}{\kappa_{2x}}
+
+Here :math:`d_{piezo}` reflects the piezo distance, :math:`d_{no tether}` reflects the calibrated surface-to-surface distance without the tether with the trap position given by :math:`x_{trap1}`.
+:math:`F_{1x}` and :math:`F_{2x}` are the forces measured on the bead and :math:`\kappa_{1x}` and :math:`\kappa_{2x}` represent the trap stiffness for each trap.
+
+We now have all the calibrations we need to do piezo tracking on our experimental data.
+
+To do this in Pylake, we set up the piezo distance calibration as follows::
+
+    piezo_calibration = lk.PiezoForceDistance(distance_calibration, baseline_1x, baseline_2x)
+
+Next, we load the data acquired in the presence of a tether::
+
+    pulling_curve = lk.File("pulling_experiment.h5")
+
+And determine the piezo distance and corrected force::
+
+    tether_length, corrected_force_1x, corrected_force_2x = piezo_calibration.force_distance(
+        pulling_curve['Trap position']['1X'], pulling_curve.force1x, pulling_curve.force2x, downsampling_factor=100
+    )
+    force_data = - corrected_force_2x
+
+Here the downsampling factor determines how much the data is downsampled prior to piezo-tracking and baseline correction.
+
+Which we can then plot::
+
+    plt.scatter(tether_length.data, force_data.data, s=1)
+    plt.xlabel('Distance [$\mu$m]')
+    plt.ylabel('Force [pN]')
+
+.. image:: figures/piezotracking/piezotracking_result.png
+
+We can compare this to the camera-based distance and raw force curve and see a clear difference::
+
+    plt.scatter(tether_length.data, force_data.data, s=1, label="corrected")
+    plt.scatter(pulling_curve.distance1.data, - (pulling_curve.downsampled_force2x.data - f2_offset), s=1, color='r', label="raw")
+    plt.xlabel('Distance [$\mu$m]')
+    plt.ylabel('Force [pN]')
+    plt.legend()
+
+.. image:: figures/piezotracking/comparison.png
```

Comment:
any reason the second curve here is red? seems a bit odd since all of the other plots use the standard blue, orange, ... mpl color cycle